setup.py

```diff
@@ -29,7 +29,8 @@
     tests_require=['pytest'],
     entry_points={
         'console_scripts': [
-            'riunet_inf = ufgsim_pkg.riunet_inf:main'
+            'riunet_inf = ufgsim_pkg.riunet_inf:main',
```

Comment:
If you removed the riunet_inf.py file, remember to remove this line as well. Otherwise it will be installed as a executable, even though it does not exist.